python/paddle/amp/grad_scaler.py

```diff
@@ -14,7 +14,7 @@
 
 from paddle.fluid.dygraph.amp import AmpScaler
 
-__all__ = ['GradScaler']
+__all__ = ['AmpScaler']
```

Comment:
verify

Reply:
GradScaler should access by paddle.amp.GradScaler in 2.0API